### PR TITLE
fix: Fix bug with producer playing video after closing dialog

### PR DESCRIPTION
**Pull request by AdmiralCoco**
- Add logic to stop rendering producer once dialog is closed

### CODE REVIEW

**Review by MichaelKovalchuk — DISMISSED** [review]

File: components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-media-file-detail.js [hunk 1]
```diff
@@ -232,7 +236,18 @@ class ContentMediaFileDetail extends SkeletonMixin(ErrorHandlingMixin(LocalizeAc
...
 
 	_renderProducer(mediaFileEntity) {
-		const closeDialog = this._refetchCaptions.bind(this, mediaFileEntity);
+		if (!this.shouldRenderProducer) {
+			return html``;
+		}
+
+		const closeDialog = function(event) {
+			// This handler captures events from both the confirm dialog as well as the fullscreen dialog
+			this._refetchCaptions(mediaFileEntity);
```
Comment (AdmiralCoco) [inline components/d2l-activity-editor/d2l-activity-content-editor/content-file/d2l-activity-content-media-file-detail.js:245]: This handler gets triggered from the confirm dialog that shows up after clicking 'Finish', as well as when closing the fullscreen dialog - refetchCaptions continues to be called in both cases

**Review by coffeexcode — DISMISSED** [review]
LGTM

**Comment by AdmiralCoco** [comment]
@coffeexcode It looks like the CI (lint) failure is unrelated to these changes, do you know what the cause might be?

**Comment by coffeexcode** [comment]
It looks like it doesn't like that the locales doesn't have the new lang terms I added in https://github.com/BrightspaceHypermediaComponents/activities/pull/2194. Not sure why that would make CI angry though. I'll poke around.

**Review by coffeexcode — APPROVED** [review]
Yay CI!

**Pull request merged**